Libplanet.Explorer.Executable/Options.cs

```diff
@@ -47,41 +48,40 @@ public class Options
 The format is a comma-separated triple of a peer's hexadecimal public key, host, and port number.
 E.g., `02ed49dbe0f2c34d9dff8335d6dd9097f7a3ef17dfb5f048382eebc7f451a50aa1,example.com,31234'.
 If omitted (default) explorer only the local blockchain store.")]
```

Comment:
This `HelpText` is still assuming it takes only a single seed.  Could you adjust the content?